adafruit_ble/__init__.py

```diff
@@ -42,7 +43,163 @@
 
 """
 
-# imports
+import _bleio
+import board
+
+from .services.core import Service
+from .advertising import Advertisement
 
 __version__ = "0.0.0-auto.0"
 __repo__ = "https://github.com/adafruit/Adafruit_CircuitPython_BLE.git"
+
+# These are internal data structures used throughout the library to recognize certain Services and
+# Advertisements.
+# pylint: disable=invalid-name
+all_services_by_name = {}
+all_services_by_uuid = {}
+known_advertisements = set()
+# pylint: enable=invalid-name
+
+def recognize_services(*service_classes):
```

Comment:
I just realized I can remove both recognize things because 1) start_scan is given a type list it can use for the scan and 2) switching to dictionary-like syntax for services with the service type as the key gives us all the info we need.